src/resources/postcss/components/full/_loader.pcss

```diff
@@ -19,7 +19,8 @@
 			animation-duration: 2.24s;
 			animation-iteration-count: infinite;
 			animation-direction: normal;
-			fill: var(--color-accent-primary-background);
+			fill: currentColor;
+			opacity: 0.07;
```

Comment:
let's set an opacity in tribe common styles and use it here. check also where it is in utilities.

Reply:
opened this https://github.com/moderntribe/tribe-common-styles/pull/75 for a new opacity value